### PR TITLE
Autogenerate updated CLI docs for 4.9

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2377

First run on August 18. Will run again after code freeze.

Previews:
* https://deploy-preview-35593--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/developer-cli-commands.html
* https://deploy-preview-35593--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/administrator-cli-commands.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@zhouying7780 I am doing an initial pull in of the latest autogenerated CLI docs. Can you take a look please?

I'll do another run after code freeze to get any late breaking changes.

Previews:
* https://deploy-preview-35593--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/developer-cli-commands.html
* https://deploy-preview-35593--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/administrator-cli-commands.html

File: modules/oc-adm-by-example-content.adoc [hunk 1]
```diff
@@ -395,11 +419,11 @@ Drain node in preparation for maintenance
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # Drain node "foo", even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.
-  $ oc adm drain foo --force
+  # Drain node "foo", even if there are pods not managed by a replication controller, replica set, job, daemon set or stateful set on it
```
Comment (EricPonvelle) [inline modules/oc-adm-by-example-content.adoc:422]: This list needs an Oxford comma after "daemon set" and before "or" per the IBM style guide.

**Review by EricPonvelle — APPROVED** [review]
/lgtm

Two super minor stylistic items, but they aren't critical if they need to wait.

**Comment by bergerhoffer** [comment]
Thanks @EricPonvelle! Since these descriptions are pulled from code, I won't be able to make any changes in this PR. But I'll make a note of them if I do any future updates to the descriptions/examples in the oc or kubectl code.

**Pull request merged**